config/auth.go

```diff
@@ -11,12 +11,12 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
-	errgo "gopkg.in/errgo.v1"
```

Comment:
I removed this dependency from this file and it had a lot of side effects, sorry.. :sweat_smile: